BuildingBlocks/src/DevelopmentKit.Identity/ValueObjects/IdentityAddress.cs

```diff
@@ -47,29 +48,39 @@ public static bool IsValid(string? stringValue)
 
         var lengthIsValid = stringValue.Length <= MAX_LENGTH;
 
-        var realm = stringValue[..3];
+        const string pattern = @"^did\:web\:(.+?)\:dids\:(.{3})(.+)$";
+        var matches = Regex.Matches(stringValue, pattern, RegexOptions.IgnoreCase);
 
-        var concatenation = Base58.Bitcoin.Decode(stringValue.AsSpan(3)).ToArray();
-        var hashedPublicKey = concatenation[..20];
-        var givenChecksum = concatenation[20..];
+        try
+        {
+            var realm = matches.First().Groups[2].Value;
```

Comment:
There will be no more `realm`. So you can remove it from the validation and generation